<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileWindow</class>
 <widget class="QDialog" name="FileWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPlainTextEdit" name="solved_system">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>291</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="copyButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Скопіювати</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Зберегти до файлу</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
